README.md

```diff
@@ -29,14 +29,39 @@ make install
 ```
 
 Install location can be controlled by passing `DESTDIR=<DIR>` before the
-`install` target. You can also control the location of the installed environment
-files on the final target with `ENV_DIR`. The default is `/boot/efi/EFI/BOOT`
-which assumes that the [EFI
+`install` target. By default, the installation is integrated with the
+`/etc/grub.d` scripts, which means you need to run `grub-install` and
+`update-grub` after installation on the platform. This normally has to be done
+while running the device, but you can use
+[mender-convert](https://github.com/mendersoftware/mender-convert) as a way to
+do this at build time.
+
+If you want to install only the tools, or only the script and environment files,
+run `make install-tools` or `make install-grub.d-boot-files`, respectively.
+
+### Standalone boot scripts
```

Comment:
I'm assuming we want to move away from this as a general rule,
in that case it should be presented in a way that it's not a first class citizen but instead a "custom do only if you know why you're doing it" case. 

Also is the following mapping correct?:
* [Installing](https://github.com/kacf/grub-mender-grubenv/tree/grub.d_integration#installing) = `grub.d flow`
* Standalone boot scripts = `grub-mender-grubenv flow`

Reply:
It should be the default, but we can not move away from it entirely, because it is still the only way to do it in Yocto, and this is unlikely to change. But still a good point about a warning, I'll add something like that, and perhaps mention that Yocto is a place where it's used also.

The mapping is correct.